tetramer_validator/templates/base.html

```diff
@@ -52,12 +52,42 @@ <h1> Tetramer Validator </h1>
 
     </form>
 
-    {% if statement and error%}
-    <p class="alert alert-danger" role="alert"> {{statement}} </p>
-    {% elif statement %}
-    <p class="alert alert-success" role="alert"> {{statement}} </p>
+
+
+    {% for error in errors%}
```

Comment:
Please use one `alert-danger` box, containing a list of errors.

Reply:
Commit 61f23fd

Reply:
A list is better. Please use `<ul>` and `<li>` instead of `<p>` and `<br/>`.

Reply:
3bdb80d8cb9d13f6a5e9051e4b675ffffc349010